migrations/v6-MIGRATION.md

```diff
@@ -0,0 +1,123 @@
+## v6 API changes
+
+### New
+
+| Name                   |
+|------------------------|
+| `PurchaseOption`       |
+| `GooglePurchaseOption` |
+| `GoogleStoreProduct`   |
+| `Price`                |
+| `PricingPhase`         |
+| `RecurrenceMode`       |
```

Comment:
We might not need this, but it won't hurt

Reply:
i think `AmazonStoreProduct` is new too

Reply:
good call